src/components/dropdown.tsx

```
@@ -0,0 +1,42 @@
+const Dropdown = ({ size = "md", items }: DropdownProps) => {
+  return (
+    <div
+      className={`absolute ${
+        size === "sm" ? "w-[81px] h-auto" : "w-[106px] h-auto"
+      } shadow-lg bg-white rounded-[12px] overflow-hidden`}
+    >
+      {items.map((item, index) => (
+        <button
+          key={`${item.label}-${index}`}
+          onClick={item.onClick}
+          className={`w-full flex justify-center items-center text-[#334155] font-pretendard font-normal ${
```

ESLint

[error] Replace `w-full·flex·justify-center·items-center·text-[#334155]·font-pretendard·font-normal` with `font-pretendard·flex·w-full·items-center·justify-center·font-normal·text-[#334155]`

(prettier/prettier)